### ISSUE TITLE
"No certificate matching..." for release build

### ISSUE THREAD

**Issue by akx** [lwouis/alt-tab-macos#192]
A minor thing: when following the instructions in docs/Contributing, the build fails since I'm not @lwouis. Perhaps the project files should somehow be amended to be identity agnostic, or the instructions changed to create a `Debug` scheme build since that works fine.

```
~/b/alt-tab-macos (master) $ pod install
...
Pod installation complete! There are 2 dependencies from the Podfile and 2 total pods installed.
~/b/alt-tab-macos (master *) $ xcodebuild -workspace alt-tab-macos.xcworkspace -scheme Release
...
error: No certificate matching 'Developer ID Application: Louis Pontoise (QXD7GW8FHY)' found: Select a different signing certificate for CODE_SIGN_IDENTITY, a team that matches your selected certificate, or switch to automatic provisioning. (in target 'alt-tab-macos' from project 'alt-tab-macos')
~/b/alt-tab-macos (master *) $
```

**Comment by lwouis** [comment]
Oh I need to update the instructions! Indeed the way I designed this is: `Debug` builds for local development/debugging/profiling and `Release` builds for production/public releases.

I think you mentioned you are able to build the project locally using the `Debug` build. I just want to make sure you're all set. Let me know if you have any question! I want collaborative work to be as smooth as possible. It's quite a challenge in the XCode world, and I worked really hard to not have magic things on my local machine that no other people have. Let me know if I missed something, or if you have an improvement you would like! 👍
